screenpipe-server/src/server.rs

```diff
@@ -289,27 +275,6 @@ pub(crate) async fn stop_device(
     }))
 }
 
-pub(crate) async fn start_recording(
```

Comment:
unfortunately restart api was kinda broken, lets re-add once we have more traction on this feat